test/parallel/test-bootstrap-modules.js

```diff
@@ -162,11 +163,6 @@ if (process.features.inspector) {
   expected.beforePreExec.add('Internal Binding inspector');
   expected.beforePreExec.add('NativeModule internal/util/inspector');
   expected.atRunTime.add('NativeModule internal/inspector_async_hook');
-
-  // This is loaded if the test is run with NODE_V8_COVERAGE.
-  if (process.env.NODE_V8_COVERAGE) {
```

Comment:
This might be removed wrongly while resolving conflicts.

Reply:
Probably, yes. I don't recall this part at all. 😅